extra/integration-testing/docker-compose.compat-2.0.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.1'
+services:
+  mender-client-2-0:
+    image: mendersoftware/mender-client-qemu:2.0
+    networks:
+       - mender
+    stdin_open: true
+    tty: true
+    privileged: true
+    environment:
+      TENANT_TOKEN: ""
+    volumes:
+      # Provision demo certificate
+      - ./cert/cert.crt:/mnt/config/server.crt
+      # Add updated setup that puts the new demo certificate in the global ca trust store
+      - ./extra/integration-testing/setup-mender-configuration.py:/setup-mender-configuration.py
+
```

Comment:
I suggest to put all the yaml files for this test in a subdir, something like `extra/integration-testing/test-compat/...`

The parent directory, `extra/integration-testing` contains files used by other parts (`extra/integration-testing/docker-compose.yml` is used by backend tests!) so I think it will look tidier to get these used by a single test in a subdir.

Reply:
sure fixed. also: we now scan the directory and get the versions from there.